lib/commands/connect/import.js

```diff
@@ -8,14 +8,16 @@ module.exports = {
     command: 'import',
     description: 'Import configuration from an export file',
     help: 'Imports the mapping configuration from a json export file',
+    args: [
+      {name: 'file', desciption: 'JSON export file name', hasValue: true}
+    ],
     flags: [
-      {name: 'resource', char: 'r', description: 'specific connection resource name', hasValue: true},
-      {name: 'file', char: 'f', desciption: 'Json export file name', hasValue:true}
+      {name: 'resource', char: 'r', description: 'specific connection resource name', hasValue: true}
     ],
     needsApp: true,
     needsAuth: true,
     run: function (context) {
-      if (!context.flags.file) {
+      if (!context.args.file) {
         console.log("You must support the file name of an export file");
```

Comment:
Actually, this won't even get touched. Now that it's a proper argument, the toolbelt ensures its presence before it even runs our code. Say goodbye!